package.json

```diff
@@ -144,19 +146,12 @@
   "config": {
     "commitizen": {
       "path": "./node_modules/cz-conventional-changelog"
-    },
-    "commitTypeMap": {
-      "docs": null,
-      "fix": "patch",
-      "perf": "patch",
-      "feat": "minor",
-      "refactor": null,
-      "release": "minor",
-      "ci": null,
-      "tests": null
     }
   },
   "release": {
-    "analyzeCommits": "@bubltechnology/customizable-commit-analyzer"
```

Comment:
I used that commit analyzer to prevent that annoying `ENOCHANGE`, I sure you are aware of that issue. Can you set custom rules so every fix/feat commits are always released no matter what.

https://github.com/semantic-release/semantic-release/issues/192#issuecomment-333328071

Reply:
Not sure what you mean. The comment you link just describes the behaviour as I would expect, and the issue is closed. Maybe the issue was fixed? I have never had any problem with an ENOCHANGE error (I have a lot of repos with semantic release set up).

Reply:
`ENOCHANGE` appears when semantic release think there are not enough changes in code to justify a new release, but actually there are. Its still there https://github.com/semantic-release/semantic-release/blob/eec4e68615c75963e06bac810b2845cc4dee02dd/index.js#L39

This is a really annoying issue but could be fixed easily by using a commit analyzer, like I used `@bubltechnology/customizable-commit-analyzer`. My goal was to have it release on `fix` / `feat` no matter what.

If you dont have any suitable suggestion than current commit analyzer, please keep current commit analyzer.

Reply:
Sorry, I don't understand. What is the issue? semantic-release is all about semantically determining _whether_ and _what_ release should happen on a build. With the default configuration, a `fix` triggers a patch release and a `feat` commit triggers a minor release. If there was only a `chore`, `ci`, `tests`, `docs` etc commit then there were no relevant changes, hence ENOCHANGE. That's how it's supposed to work.

Reply:
This is the default implementation: https://github.com/semantic-release/commit-analyzer/

Reply:
> With the default configuration, a fix triggers a patch release and a feat commit triggers a minor release. 

This is not true for all cases, sometimes semantic release thinks there are not enough changes in code which could justify a new release (even if commit is `fix / feat`), hence `ENOCHANGE`.

Please keep this commit analyzer implementation (or use any official method) so all `fix / feat` commits are releases no matter what

Reply:
I don't see any evidence for why that would be the behaviour. If it is like that, it is a bug (that should be reported and fixed). But the issue you linked is closed, the default commit analyzer is depended on by hundreds on projects, got a lot of updates since we added semantic-release and has 100% test coverage. 

`@bubltechnology/customizable-commit-analyzer` on the other hand doesn't even have a stable release, way fewer dependents, no integration tests and has not any more customizability than the official implementation. It's also not even compatible with semantic-release v11, as v11 requires plugins to return a Promise, while that plugin still tries to call a callback.

I have high confidence that the official implementation works correctly.

Reply:
Ok lets merge this. If it worked as we expected it to (release on every fix / feat commit) then great, otherwise we can figure something out.